libsplinter/src/biome/key_management/key_store.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WI()HOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::{KeyStore, KeyStoreError, SplinterKey};
```

Comment:
This relationship is awkward. It would make more sense to have something like:

biome::key_management::store::KeyStore
biome::key_management::error::KeyStoreError
biome::key_management::store::postgres::PostgresKeyStore

All of these could, if desired, be re-used "pub use ..." in biome::key_management to make them available at the top level.